### DISCUSSION TITLE
Is this a memory database?

### DISCUSSION THREAD

**Discussion by ImSingee** [tinyplex/tinybase#121] [Q&A]
I do a little try on tinybase and read some of the documents, I really want to know if what I'm thinking is correct:

1. tinybase is a memory database, it loads everything in memory on start
2. The index is not persistent, it's built when load data

Am I correct?

**Comment by jamesgpearce** [comment]
Correct on both counts! But there are lots of ways to persist the data, of course. Good luck!
